src/commands/remove-demo-markup.ts

```diff
@@ -0,0 +1,117 @@
+import { GluegunToolbox } from "gluegun"
```

Comment:
Quite a bit of this is duplicated from `./src/commands/remove-demo.ts`

I'd be happy to refactor `remove-demo` to include another switch for `--markup` or something similar and just conditionally check if we should remove files or remove markup.

Thoughts?

Reply:
Maybe `--only-markup` ?

Reply:
I think we could also migrate a lot of this functionality into the `demo` tool as new methods. 
 
```ts
const update = (filePaths: string[]): Promise<{ path: string; comments: string[] }>[] => {
  // demoCommentResults block
  // ...
  return demoCommentResults.sort((a, b) => {
    if (a.status === "fulfilled" && b.status === "fulfilled") {
      return a.value.path.localeCompare(b.value.path)
    }
    return 0
  })
}

// ...

const find = (targetDir: string, matching?: string[]) => {
  const MATCHING_GLOBS = [
    "!**/.DS_Store",
    "!**/.expo{,/**}",
    "!**/.git{,/**}",
    "!**/.vscode{,/**}",
    "!**/node_modules{,/**}",
    "!**/ios/build{,/**}",
    "!**/ios/Pods{,/**}",
    "!**/ios/*.xcworkspace{,/**}",
    "!**/android/build{,/**}",
    "!**/android/app/build{,/**}",
  ]

  const filePaths = filesystem
    .cwd(targetDir)
    .find({
      matching: matching ?? MATCHING_GLOBS,
      recursive: true,
      files: true,
      directories: false,
    })
    .map((path) => pathlib.join(targetDir, path))
  return filePaths
}

export const demo = {
  // ...
  update,
  find,
} as const
```

Reply:
@joshuayoes could have sworn I wrote a comment earlier 🙃 . Would we need to pass in filesystem and patching here to make this work as well? Is that something we want to do?

Reply:
Yeah, we can import these directly. I've talked with @jamonholmgren about this, and we don't really utilize dependency injection in our tests, so we can import them directly from toolbox

Reply:
```
import { filesystem, patching } from "gluegun"
```